### DISCUSSION TITLE
Some way to choose Bing/Sydney conversation style

### DISCUSSION THREAD

**Discussion by antonme** [danny-avila/LibreChat#65] [Feature Requests & Suggestions]
It would be great if there would be some way to choose between different Bing/Sydney conversation styles, as it feels quite a much different thing in Creative or Precise modes. Maybe something like creating CustomGPT variant, or just by multiplying default Bing and Sydney x3 by showing Bing creative, Bing balanced and Bing precise as different models

**Comment by danny-avila** [comment]
This is a planned update! It should work like it does on bing.com, you will have the options on a blank conversation (or if the model is switched)

**Reply by antonme** [reply]
That is really great! Maybe it would be a good moment to also add some way to edit Sydney's system prompt – right now it is hardcoded into the API server, but every so often it is really beneficial to change it a bit.

**Reply by danny-avila** [reply]
this is being added to waylaidwanderer's project and when it is, i'll be able to implement it. Seems like it will be the only way to make Sydney work atm

**Reply by danny-avila** [reply]
https://github.com/waylaidwanderer/node-chatgpt-api/discussions/187#discussioncomment-5345724
